l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -123,12 +123,12 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
 
     /// @notice To set stmDeploymetTracker, only Owner. Not done in initialize, as
     /// the order of deployment is Bridgehub, Shared bridge, and then we call this
-    function setSTMDeployer(ISTMDeploymentTracker _stmDeployer) external onlyOwner {
-        stmDeployer = _stmDeployer;
-    }
+    // function setSTMDeployer(ISTMDeploymentTracker _stmDeployer) external onlyOwner {
+    //     stmDeployer = _stmDeployer;
+    // }
 
-    /// @notice To set shared bridge, only Owner. Not done in initialize, as
-    /// the order of deployment is Bridgehub, Shared bridge, and then we call this
+    /// @notice To set addresses, only Owner. Not done in initialize, as
```

Comment:
let's have a clarification of sort "addresses of some of the ecosystem contracts"